source/_integrations/inels.markdown

```diff
@@ -0,0 +1,72 @@
+---
+title: iNELS
+description: Instructions on how to integrate iNELS with Home Assistant.
+ha_category:
+  - Binary Sensor
+  - Button
+  - Climate
+  - Cover
+  - Light
+  - Select
+  - Sensor
+  - Switch
```

Comment:
An initial PR of integration can only bring in a single platform.
Most of these categories seem, therefore, not be fitting the linked parent PR in the Core repository.